src/card-button.css

```diff
@@ -0,0 +1,24 @@
+.card-button {
+  margin-top: 16px;
+  display: block;
+}
```

Comment:
As I said before, if it is a BEM block you shouldn't use margins here, make button as an card block element and that's it